sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -210,6 +210,17 @@ object Window {
     spec.rangeBetween(start, end)
   }
 
+  /**
+   * @param start boundary start, inclusive. The frame is unbounded if the expression is
```

Comment:
we should copy-paste the document from `rangeBetween(start: Long, end: Long)` to here, and replace `long values` with `literal values`. We should also update the examle to show how to use special range boundary.